lib/recurly/resource.rb

```diff
@@ -1000,6 +1000,7 @@ def marshal_dump
         @destroyed,
         @uri,
         @href,
+        @type,
```

Comment:
To prevent problems with upgrading to this change, we need to move the declaration for `@type` to the end of the array here.